velox/functions/prestosql/window/tests/LeadLagTest.cpp

```diff
@@ -323,5 +323,62 @@ VELOX_INSTANTIATE_TEST_SUITE_P(
     LeadLagTest,
     ::testing::Values("lead"));
 
+// DuckDB has errors in IGNORE NULLS logic for empty
```

Comment:
Would you create a GitHub issues and reference it here?

Reply:
Didn't quite follow. Do you mean an issue for DuckDB ?

Reply:
It sounds like there is a bug in DuckDB, hence, I suggested to report it by creating a GitHub issue in DuckDB repo. They may just fix it.

Reply:
Alright. Created https://github.com/duckdb/duckdb/issues/8217